libs/tdigest/build.gradle

```diff
@@ -22,6 +22,8 @@ apply plugin: 'elasticsearch.build'
 apply plugin: 'elasticsearch.publish'
 
 dependencies {
+  api project(':libs:elasticsearch-core')
```

Comment:
For `Releasable`

Reply:
I really wonder if we really need this abstraction here. Releasable is very much an Elasticsearch concept and it feels it does not belong to this library.

I like dependency free libraries, but if folks are ok with this, I am ok too.

Reply:
_Need_, not really. But it also helps in tests, as we can use Releasables methods.

I'll be adding a dep on lucene-core in a next PR for `Accountable` (Which is similar). We can discuss it there, if it sounds weird (To merge this PR for now)

Reply:
Sounds good,. We can asses in the following PR if its worthy to remove the dependency by adding a few more lines of code,